Remove a test which fails when run as root.


--- mu4e-1.12.12/lib/tests/test-mu-store.cc	2025-07-19 07:03:38.000000000 +0200
+++ mu4e-1.12.12/lib/tests/test-mu-store.cc	2025-09-08 16:15:32.457387763 +0200
@@ -580,12 +580,6 @@
 		const auto store = Store::make("/root/non-existent-path/12345");
 		g_assert_false(!!store);
 	}
-
-	{
-		const auto store = Store::make_new("/../../root/non-existent-path/12345",
-						   "/../../root/non-existent-path/54321");
-		g_assert_false(!!store);
-	}
 }
